src/utils/constants.js

```diff
@@ -0,0 +1,10 @@
+const BRANCH = process.env.BRANCH || 'dev';
```

Comment:
It doesn't quite make sense to keep this variable of a `string` type as it is a boolean condition, it is either production or not. Besides, this variable can be confusing - it will match of course the branch name in prod, but I doubt all other branches are named `dev` 😉

Reply:
Let's rename it everywhere to `IS_PRODUCTION`